app/helpers/orchid/application_helper.rb

```diff
@@ -162,6 +162,55 @@ def prefix_path(path, *args, **kwargs)
     end
   end
 
+  # Render section overrides and localized partials
+  # Partial name does not include the locale, underscore, or extensions
+  # Example:
+  #   render_overridable("explore/partials", "index")
+  #   Looks for in order:
+  #     If @section defined & locale == "es", "@section/_index_es.html.erb"
+  #     If @section defined & no locale, "@section/_index.html.erb"
+  #     If no @section & locale == "es", "explore/partials/_index_es.html.erb"
+  #     If no @section or locale, "explore/partials/_index.html.erb"
+  #  If no partial found, render an error message with missing partial path
+  def render_overridable(path="", partial="", **kwargs)
```

Comment:
I am kind of confused in this section about how and why path is being assigned / unassigned, I think maybe when you're back if you can talk me through it that would be helpful!

Reply:
I took a stab at improving the comments to make this easier to understand. Let me know if this makes sense: https://github.com/CDRH/orchid/pull/133/commits/31ac07612791545428f499861a335c4e8618e81c

It's been a while since we discussed this, so hopefully it will be like coming at this with fresh eyes again and will be a good test for whether it's really clear or not :smile: